src/lp_data/Highs.cpp

```diff
@@ -2874,14 +2873,12 @@ HighsStatus Highs::callRunPostsolve(const HighsSolution& solution,
     // Set up the iteration count and timing records so that
     // adding the corresponding values after callSolveLp gives
     // difference
-    postsolve_iteration_count = -info_.simplex_iteration_count;
```

Comment:
Shouldn't the preceding comment ("Set up the iteration count...") also be updated?

Reply:
Of course